config/config/sc-config.yaml

```diff
@@ -154,9 +171,21 @@ elasticsearch:
       limits:
         memory: 1024Mi
         cpu: 1
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+        - weight: 1
+          podAffinityTerm:
+            topologyKey: "kubernetes.io/hostname"
+            labelSelector:
+              matchLabels:
+                role: master
```

Comment:
Do we always want this? I'm thinking that we may want `requiredDuringSchedulingIgnoredDuringExecution` for prod and skip it completely for dev. It is fine like this also I'm just trying to understand what this gives us. As long as it is `preferred` the scheduler will still put the pods together if some other pods were scheduled earlier and "filled" the other nodes.
Same for the other affinities.

Reply:
Sure we can can leave it empty for dev. Don't want to update the prod flavor with `requiredDuringSchedulingIgnoredDuringExecution` because i don't have time to configure and construct the entire setup so that it actually can schedule all pods.

So while i agree that we should spend more time on the flavors, this pr was more or less just meant as an entrypoint inorder to acutally be able to configure this stuff, hence why all values you see in the config now are what is already being used.

Reply:
I see, thanks for the clarification. Let's keep it like this :slightly_smiling_face:

Reply:
Oki :;)